contracts/test/helpers/ACLHelper.sol

```diff
@@ -15,27 +15,27 @@ contract ACLHelper {
 
 
 contract AcceptOracle is IACLOracle {
-    function canPerform(address, address, bytes32, uint256[]) external view returns (bool) {
+    function canPerform(address, address, address, bytes32, uint256[]) external view returns (bool) {
```

Comment:
I wonder, does it make sense to assign an ACL Oracle with anything else than `ANY_ENTITY`? Because otherwise we could keep only one param and send the real sender instead of `ANY_ENTITY` in `_hasPermission`.

Reply:
> does it make sense to assign an ACL Oracle with anything else than `ANY_ENTITY`?

Probably it doesn't, but I wouldn't assume that.

Reply:
Contrived, but you could assign a specific entity (e.g. an Agent) from not performing something via the ACL Oracle.

This could, for example, be used to disallow the AN DAO from changing court values too steeply.

Reply:
I don’t get it: so would you assign the permission to the Agent instead of `ANY_ENTITY`? Then only the agent would be able to call the function protected by the ACL Oracle, right?

Reply:
Yes, exactly.

Reply:
But if you assign the role to a particular entity (Agent for instance), what do you want the new address param for?

Reply:
Right, attending to the initial question:

> Because otherwise we could keep only one param and send the real sender 

It is contrived, but I consider the old behaviour as "not a bug" specifically because your ACL Oracle could behave differently if it was defined to be completely open for `ANY_ADDRESS` vs. specific to an address.

Reply:
Hm, that makes ACL Oracles even more complex and harder to understand.

Reply:
Agreed—in this case, it was a couple of different concerns:

- This information about the `ANY_ENTITY` grantee was available in the old version
- We needed a different function signature so any combination of ACLv1, ACLv2, ACLOracleV1, and ACLOracleV2 could work together (assuming the app implemented both)